src/Microsoft.CodeAnalysis.Razor/Microsoft.CodeAnalysis.Razor.csproj

```diff
@@ -14,4 +14,8 @@
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="$(MicrosoftCodeAnalysisCSharpPackageVersion)" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net46' ">
```

Comment:
Looks like we shipped a 2.0 version of this package with the NS.Library version. Can we get a confirmation from @Eilon \ @muratg \ @mkArtakMSFT  that this is ok to remove?

Reply:
Here's the 2.0.1 package for reference: https://www.nuget.org/packages/Microsoft.CodeAnalysis.Razor/2.0.2

Reply:
The thinking here is that our packages should never have NETStandard.Library in their generated nuspec. This was only added to our .NET Framework TFMs as an accident, and should never have been in. The package ref to NETStandard.Library is unnecessary for .NET Framework projects.

Reply:
I'm all for it. Just making sure the powers to be approve of this 😄

Reply:
So we're dropping a "bogus" reference? I can live with that.

Reply:
Yes agreed. Approved. Unless I'm hugely misunderstanding something, this shouldn't actually break anyone.